proof/refine/AARCH64/ArchAcc_R.thy

```diff
@@ -840,11 +844,13 @@ next
     apply (simp add: pt_bits_def)
     apply (prop_tac "level_type (nlevel + 1) = NormalPT_T")
      apply (drule max_pt_level_enum)
-     apply (auto simp: level_defs split: if_split_asm)[1]
+     (* Unfolding here, because the level_enum only produces a contradiction in one of the branches
+        when PA_40 is set. *)
```

Comment:
If you're making the other comment change.
```suggestion
     (* Unfolding config_ARM_PA_SIZE_BITS_40_def here, because the level_enum only produces a contradiction in one of the branches
        when PA_40 is set. *)
```